_posts/2019-09-20-lambda-to-openfaas.md

```diff
@@ -0,0 +1,330 @@
+---
+title: "Migrate Your AWS Lambda Functions to OpenFaaS"
+description: Burton explains the steps for migrating an existing AWS Lambda function to OpenFaaS
+date: 2019-09-10
+image: /images/lambda-to-openfaas/birds.jpg
+categories:
+  - Lambda
+  - tutorial
+  - examples
+author_staff_member: burton
+dark_background: false
+---
+
+AWS introduced the world to the idea of serverless architecture back in 2014 with the announcement of Lambda. The idea was that you could upload a zip file containing your code logic, and AWS would manage the infrastructure behind the scenes, billing you only for the time the code was being executed.
```

Comment:
"The idea was that you could upload a zip file containing your code logic, and AWS would manage the infrastructure behind the scenes, billing you only for the time the code was being executed."

Break the sentence up using multiple sentences or and. I.e. stop at "behind the scenes" and start a new sentence

Reply:
Done